inputs/france/2010_stop_rep_1970-2160.xml

```diff
@@ -0,0 +1,2285 @@
+<!-- Historic, France, stops reprocessing at 2020 -->
+<simulation>
+  <control>
+    <duration>2280</duration>
+    <startmonth>1 </startmonth>
+    <startyear>1970 </startyear>
+    <decay>lazy</decay>
+  </control>
+
+  <archetypes>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Source</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Sink</name>
+    </spec>
+    <spec>
+    	<lib>cycamore</lib>
+    	<name>Reactor</name>
+    </spec>
+    <spec>
+        <lib>cycamore</lib>
+        <name>Mixer</name>
+    </spec>
+    <spec>
+      <lib>agents</lib>
+      <name>NullRegion</name>
+    </spec>
+    <spec>
+      <lib>agents</lib>
+      <name>NullInst</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Separations</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>DeployInst</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Separations</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>Enrichment</name>
+    </spec>
+    <spec>
+      <lib>cycamore</lib>
+      <name>FuelFab</name>
+    </spec>
+  </archetypes>
+
+
+<facility>
+    <name>mixer</name>
+    <config>
+      <Mixer>
+        <in_streams>
+          <stream>
+            <info>
+              <mixing_ratio>.09</mixing_ratio>
+              <buf_size>1e+299</buf_size>
+            </info>
+            <commodities>
+              <item>
+                <commodity>Pu</commodity>
+                <pref>1.0</pref>
+              </item>
+            </commodities>
+          </stream>
+          <stream>
+            <info>
+              <mixing_ratio>.91</mixing_ratio>
+              <buf_size>1e+299</buf_size>
+            </info>
+            <commodities>
+              <item>
+                <commodity>tailings</commodity>
+                <pref>1.0</pref>
+              </item>
+            </commodities>
+          </stream>
+        </in_streams>
+        <out_commod>mox</out_commod>
+        <out_buf_size>1e+299</out_buf_size>
+        <throughput>1e+299</throughput>
+      </Mixer>
+    </config>
+</facility>
+
+
+<!-- La Hague Model from world-nulcear.org -->
+  <facility>
+    <name>reprocessing</name>
+    <config>
+      <Separations>
+         <feed_commods>   <val>uox_waste</val> </feed_commods>
+         <feed_commod_prefs> <val>1.0</val> </feed_commod_prefs>
+         <feed_recipe>uox_used_fuel_recipe</feed_recipe>
+         <feedbuf_size>1e100</feedbuf_size>
+         <throughput>1e100</throughput>
+         <leftover_commod>reprocess_waste</leftover_commod>
+         <leftoverbuf_size>1e+290</leftoverbuf_size>
+         <streams>
+          <item>
+            <commod>Pu</commod>
+            <info>
+              <buf_size>1e100</buf_size>
+              <efficiencies>
+                <item>
+                  <comp>Pu</comp> <eff>.998</eff>
```

Comment:
I got my information from page 15 of [this AREVA presentation](https://www.iaea.org/INPRO/2nd_Dialogue_Forum/AIEA_INPRO_meeting-Multilateral_aspects_of_reprocessing_and_recycling-D_GRENECHE.pdf).

Obviously this is not the most reliable (or quotable) source. Do you have another source? Thanks!